examples/meter/css/meter.css

```diff
@@ -7,9 +7,10 @@
   width: 200px;
 }
 
-#fill {
+.fill {
   box-sizing: border-box;
   height: 100%;
   border-radius: 3px;
-  border: 20px solid black;
+  background-color: black;
+  outline: 3px solid black;
```

Comment:
Out of interest: is there a reason to use `outline` rather than `border` here?

Reply:
I think I switched it back to border, and reduced it to 2px